.github/workflows/release.yaml

```diff
@@ -27,65 +27,77 @@ jobs:
     needs: [version-match]
     strategy:
       matrix:
-        integration-test-script: [
-          'integration-test.sh',
-          'preconfig-integration-test.sh',
-          'cosign-integration-test.sh',
-          'deployment-integration-test.sh']
+        integration-test-arg: ["regular", "cosign", "deployment", "pre-config"]
```

Comment:
In comment in cicd.yaml it was named `preconfig`, here `pre-config`. Consistency